README.md

```diff
@@ -177,6 +177,7 @@ python finetune.py \
   --num-epochs '2' \
   --eval-file 'user_oriented_instructions.jsonl' \
   --eval-limit '5'
+  --adam-betas 0.99 0.999
```

Comment:
Did this run? I had to specify a list of arguments by passing it multiple times: --adam-betas 0.99 --adam-betas 0.999

Reply:
yes it is running. With tuple it is working

Reply:
seems like Tuple is different from List in typer

Reply:
Ah that makes life so much easier